lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -0,0 +1,267 @@
+import 'dart:ui';
+
+import 'package:equatable/equatable.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:fl_chart/src/utils/lerp.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+typedef GetTitleByIndexFunction = String Function(int index);
+
+class RadarChartData extends BaseChartData {
+  //ToDo(payam) : document data classes
+  RadarChartData({
+    this.dataSets = const [],
```

Comment:
We have a convention for default parameters in constructors. please follow it by checking other data classes.
It works like below:

```
class MyClass {
  final String firstName;

  final String lastName;

  MyClass({
    String firstName,
    String lastName,
  })  : firstName = firstName ?? 'defaultFirstName',
        lastName = lastName ?? 'defaultLastName';
}
```